content/en/apps/guides/debugging/secure-sharing-of-developer-instance.md

```diff
@@ -19,22 +19,24 @@ When using a local [development environment](https://github.com/medic/cht-core/b
 
 [<img src="/apps/guides/debugging/images/SSH.tunnel.diagram.svg" width=100% height=100%>](/apps/guides/debugging/images/SSH.tunnel.diagram.svg)
 
-Once you have this web server set up, you many continue to use when ever you want by simply reconnecting to it via the secure tunnel.
-
-## Prereqs
-This guide assumes:
-* You have a local dev instance set up of cht-core https://github.com/medic/cht-core/blob/master/DEVELOPMENT.md
-* You have medic-android repo checked out and can compile the android app from scratch https://github.com/medic/medic-android/
-* You  have the generic Medic Mobile app installed on your android device https://play.google.com/store/apps/details?id=org.medicmobile.webapp.mobile&hl=en_US
-* You have a server running Ubuntu 18.04 or later with a public IP and a DNS entry that you can SSH into and have sudo on
+Once you have this web server set up, you may continue to use it whenever you want by simply reconnecting to it via the secure tunnel.
+
+## Prerequisites 
+
+This guide assume:
+* You have a local [dev instance](https://github.com/medic/cht-core/blob/master/DEVELOPMENT.md) set up of cht-core 
+* You  have the [generic Medic Mobile app](https://play.google.com/store/apps/details?id=org.medicmobile.webapp.mobile&hl=en_US) installed on your android device. This version allows you to enter a custom CHT URL on first run.
+* You have an Ubuntu >18.04 server with a public IP and a DNS entry that you can SSH into and have sudo on
 * You have apache >2.4.29 installed on the Ubuntu server and can add a new vhost to it, including an SSL cert. (nginx could be used instead as well, but not covered here)
 * You have certbot installed from letsencrypt.org
 
 For reference, any of the cheap servers out there (Digital Ocean has a $5/mo server https://digitalocean.com/) will enable you to do this. 
```

Comment:
Worth changing the "enable you to do this" part as I am not sure what this references.

Reply:
should be fixed in 9cc010c